### PR TITLE
[MODORDERS-890] - Change instance connection test added

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
## Purpose
[https://github.com/folio-org/mod-orders/pull/751](PR with updates)
_Describe the purpose of the pull request._

## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Comment by Dmitriy-Butramyou** [comment]
image

File: acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature [hunk 1]
```diff
@@ -32,6 +32,7 @@ Feature: mod-orders integration tests
       | 'orders.item.unopen'                   |
       | 'invoice.all'                          |
       | 'audit.all'                            |
+      | 'inventory-storage.holdings.item.post' |
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature:35]: Should it be in module permissions instead?

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/poline_change_instance_connection.feature [hunk 2]
```diff
@@ -0,0 +1,149 @@
+@parallel=false
+# for https://issues.folio.org/browse/MODORDERS-890
+Feature: poline_change_instance_connection
```
Comment (Abdulkhakimov) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/poline_change_instance_connection.feature:3]: Please rename feature name to more appropriate

File: acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature (hunk 1 above)
Comment (Dmitriy-Butramyou) [reply acquisitions/src/main/resources/thunderjet/mod-orders/orders-junit.feature]: I removed this permission from here and added it to the patch method from mod-orders.

**Review by grabsefx — APPROVED** [review]

**Pull request merged**